src/scene.cpp

```diff
@@ -34,6 +34,8 @@
 #endif
 
 
+constexpr int Scene::kStartGameDelayFrames;
+constexpr int Scene::kReturnTitleDelayFrames;
```

Comment:
Why are these needed? These should get pulled in from the header.

Reply:
static data members need definitions. Without them you'll likely get linker errors in debug builds as unoptimized code is trying to find the address of these constants.